selectel/mks.go

```diff
@@ -199,10 +199,61 @@ func mksClusterV1GetLatestPatchVersions(ctx context.Context, client *v1.ServiceC
 	return result, nil
 }
 
+func mksClusterV1GetLatestMinorVersion(ctx context.Context, client *v1.ServiceClient) (string, error) {
+	kubeVersions, _, err := kubeversion.List(ctx, client)
+	if err != nil {
+		return "", err
+	}
+
+	versions := map[string]struct{}{}
```

Comment:
It seems that we can use `parseMKSKubeVersionsV1Latest` function here instead, because it does the same stuff

Reply:
Ok, i agree, this method is unnecessary. Also I have added doc string for  `parseMKSKubeVersionsV1Latest` to make it more obvious.